src/course-team/add-user-form/AddUserForm.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import {
+  Button,
+  Form,
+  ActionRow,
+} from '@edx/paragon';
+
+import { Formik } from 'formik';
+import messages from './messages';
+import FormikControl from '../../generic/FormikControl';
+
+const AddUserForm = ({ onSubmit, onCancel }) => {
+  const intl = useIntl();
+
+  return (
+    <div className="add-user-form" data-testid="add-user-form">
+      <Formik
+        initialValues={{ email: '' }}
+        onSubmit={onSubmit}
+        validateOnBlur
+      >
+        {({ handleSubmit, values }) => (
+          <>
+            <Form.Group size="sm" className="form-field">
+              <h3 className="form-title">{intl.formatMessage(messages.formTitle)}</h3>
+              <Form.Label size="sm" className="form-label">
+                {intl.formatMessage(messages.formLabel)}
+              </Form.Label>
+              <FormikControl
+                name="email"
+                value={values.email}
+                withErrorText={false}
+                placeholder="example: username@domain.com"
```

Comment:
move it to `constants.js` please on the same root

Reply:
Done